patch_docs.py

```
@@ -0,0 +1,103 @@
+import glob
+import re
+
+files = glob.glob('server/routes/*.ts')
+
+def generate_openapi_doc(method, path):
+    # Convert path params like :id to {id}
+    openapi_path = re.sub(r':([a-zA-Z0-9_]+)', r'{\1}', path)
+    
+    # Determine basic tag
+    tag = "General"
+    if "admin" in path: tag = "Admin"
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)